packages/stryker-mocha-runner/src/MochaOptionsLoader.ts

```diff
@@ -7,18 +7,28 @@ import MochaRunnerOptions, { mochaOptionsKey } from './MochaRunnerOptions';
 export default class MochaOptionsLoader {
 
   private readonly log = getLogger(MochaOptionsLoader.name);
+  private readonly DEFAULT_MOCHA_OPTS = 'test/mocha.opts';
 
   public load(config: StrykerOptions): MochaRunnerOptions {
     const mochaOptions = Object.assign({}, config[mochaOptionsKey]) as MochaRunnerOptions;
+    let optsFileName = path.resolve(this.DEFAULT_MOCHA_OPTS);
+
     if (mochaOptions && mochaOptions.opts) {
```

Comment:
I notice that we always check if we have `mochaOptions`, but there should always be an object, right? We define it on line 13.